config/production.yaml.example

```diff
@@ -66,6 +66,13 @@ redis:
   auth: null
   db: 0
 
+# redis_sentinel:
```

Comment:
Can we put this key as `sentinel` inside `redis` key? With a `enabled` key to ensure we want to enable sentinel or not. We could also reuse `redis.auth` instead of redefining `redis_sentinel.auth`

Reply:
But they are mutually exclusive no?

Reply:
Then what do you think of a `mode: 'sentinel' | 'redis'` inside the `redis` object? So it's clearer for the admin that it needs to provide sentinel auth/configuration while still keeping the exclusive thing inside the peertube code? We can imagine in the future having a 'cluster' mode too?